docs/querying/sql-window-functions.md

```diff
@@ -88,40 +89,181 @@ You can use the OVER clause to treat other Druid aggregation functions as window
 
 Window functions support aliasing.
 
-## Define a window with the OVER clause
+## Window function syntax
+
+In general, window functions in Druid use the following syntax:
+
+
+```sql
+window_function() OVER (
+  [PARTITION BY partitioning expression]
+  [ORDER BY order expression]
+  [[ROWS, RANGE] BETWEEN range start AND range end])
+FROM table
+GROUP BY dimensions
+```
+
+```sql
+window_function() OVER w
+FROM table
+WINDOW w AS ([PARTITION BY partitioning expression] [ORDER BY order expression]
+  [[ROWS, RANGE] BETWEEN range start AND range end])
+GROUP BY dimensions
+```
 
 The OVER clause defines the query windows for window functions as follows:
-- PARTITION BY indicates the dimension that defines the rows within the window
-- ORDER BY specifies the order of the rows within the windows.
+- PARTITION BY indicates the dimension that defines window boundaries
+- ORDER BY specifies the order of the rows within the windows
+
+An empty OVER clause or the absence of a PARTITION BY clause indicates that all data belongs to a single window.
+
+In the following example, the following OVER clause example sets the window dimension to `channel` and orders the results by the absolute value of `delta` ascending:
+
+```sql
+...
+RANK() OVER (PARTITION BY channel ORDER BY ABS(delta) ASC)
+...
+```
+
+Window frames, set in ROWS and RANGE expressions, limit the set of rows used for the windowed aggregation.
+
+ROWS and RANGE accept the following values for `range start` and `range end`:
+- UNBOUND PRECEDING: from the beginning of the window as ordered by the order expression
+- _N_ ROWS PRECEDING: _N_ rows before the current row as ordered by the order expression
+- CURRENT ROW: the current row
+- _N_ ROWS FOLLOWING: _N_ rows after the current row as ordered by the order expression
+- _UNBOUNDED FOLLOWING_: to the end of the window as ordered by the order expression
```

Comment:
```suggestion
- UNBOUNDED FOLLOWING: to the end of the window as ordered by the order expression
```

Reply:
Note that this one says UNBOUNDED but line 131 says UNBOUND. Should they be the same or is this intentional?